### DISCUSSION TITLE
exe应用程序打不开，双击进入闪退

### DISCUSSION THREAD

**Discussion by GKDSSS** [mikumifa/biliTickerBuy#417] [提问专区] [answered]
https://github.com/user-attachments/assets/6ec8a04f-12eb-402d-9d22-9510749f22d7

**Comment by xmbhjQAQ** [comment]
使用的是最新版本的吗，最新版本构建好像缺少一个gradioCalendar的依赖，可以自己去补一下，或者退一个版本等待修复

**Reply by GKDSSS** [reply]
是的，2.8.8

**Reply by GKDSSS** [reply, marked as answer]
pacth_1不是补丁版吗，等待修复吧

**Comment by mikumifa** [comment]
这个打包问题现在还没解决，pyinstall问题

**Comment by wheatnobeer** [comment]
请看Issue#418能否解决这个问题

**Comment by mikumifa** [comment]
Fine 我加一个公告提示一下 @wheatnobeer
